vyper/venom/README.md

```diff
@@ -348,32 +348,32 @@ Assembly can be inspected with `-f asm`, whereas an opcode view of the final byt
   - No operation, does nothing.
 - `offset`
   - ```
-    %2 = offset %1 label1
+    %2 = offset %1 @label1
   - Similar to `add`, but takes a label as the second argument. If the first argument is a literal, the addition will get optimized at assembly time.
 
 ### Jump instructions
 
 - `jmp`
   - ```
-    jmp label
+    jmp @label
     ```
   - Unconditional jump to code denoted by given `label`.
   - Translates to `label JUMP`.
 - `jnz`
    - ```
-     jnz label1, label2, op
+     jnz op, @label1, @label2
```

Comment:
```suggestion
     jnz %op, @label1, @label2
```

Reply:
Yeah that was my thinking also, I tried to make sure that was consistent when looking over the readme